src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// Write code here
+/* eslint-disable */
+import React from 'react';
+import '../../App.scss';
+
+export default function Person(props) {
+  const personPartner = props.sex === 'm' ? 'wife' : 'husband';
```

Comment:
too much of `props.`
use de-structuring
`const { sex, isMarried, ... } = props;`
and variables without `props.` prefix